### PR TITLE
8330066: HeapDumpPath and HeapDumpGzipLevel VM options do not mention HeapDumpBeforeFullGC and HeapDumpAfterFullGC

### PR DESCRIPTION
**Pull request by alexmenkov**
The fix updates descriptions of `HeapDumpPath`/`HeapDumpGzipLevel` and `HeapDumpBeforeFullGC`/`HeapDumpAfterFullGC`/`HeapDumpOnOutOfMemoryError` VM options

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8330066](https://bugs.openjdk.org/browse/JDK-8330066): HeapDumpPath and HeapDumpGzipLevel VM options do not mention HeapDumpBeforeFullGC and HeapDumpAfterFullGC (**Bug** - P4)

### Reviewers
 * [Chris Plummer](https://openjdk.org/census#cjplummer) (@plummercj - **Reviewer**)
 * [David Holmes](https://openjdk.org/census#dholmes) (@dholmes-ora - **Reviewer**)
 * [Yi Yang](https://openjdk.org/census#yyang) (@y1yang0 - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/19224/head:pull/19224` \
`$ git checkout pull/19224`

Update a local copy of the PR: \
`$ git checkout pull/19224` \
`$ git pull https://git.openjdk.org/jdk.git pull/19224/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 19224`

View PR using the GUI difftool: \
`$ git pr show -t 19224`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/19224.diff">https://git.openjdk.org/jdk/pull/19224.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/19224#issuecomment-2109071079)

### CODE REVIEW

File: src/hotspot/share/runtime/globals.hpp [hunk 1]
```diff
@@ -555,15 +555,18 @@ const int ObjectAlignmentInBytes = 8;
                                                                             \
   product(bool, HeapDumpOnOutOfMemoryError, false, MANAGEABLE,              \
           "Dump heap to file when java.lang.OutOfMemoryError is thrown "    \
-          "from JVM")                                                       \
+          "from JVM "                                                       \
+          "(also see HeapDumpPath, HeapDumpGzipLevel)")                    \
```
Comment (y1yang0) [inline src/hotspot/share/runtime/globals.hpp:559]: Bad format, otherwise looks good.

**Review by plummercj — APPROVED** [review]

**Review by dholmes-ora — APPROVED** [review]
Updates look good.

FWIW the Java command reference doesn't mention the before/after full GC flags either.

**Review by y1yang0 — APPROVED** [review]

**Pull request closed without merging**